hotel/tests/test_attendee_properties.py

```diff
@@ -25,6 +25,30 @@ def test_hotel_shifts_required_preshifts(monkeypatch):
     assert not Attendee().hotel_shifts_required
 
 
+@pytest.mark.parametrize('first,last,legal,expected', [
+    ('First', 'Last', None, 'Last'),
+    ('First', 'Last', 'First Last', 'Last'),
+    ('First', 'Last', 'First Middle Last', 'Last'),
+    ('CRAZY', 'Last', 'First Last', 'Last'),
+    ('CRAZY', 'CRAZY', 'First Last', 'Last'),
+    ('CRAZY', 'CRAZY', 'First Middle Last', 'Middle Last'),
+])
+def test_legal_last_name(first, last, legal, expected):
+    assert expected == Attendee(first_name=first, last_name=last, legal_name=legal).legal_last_name
+
+
+@pytest.mark.parametrize('first,last,legal,expected', [
+    ('First', 'Last', None, 'First'),
+    ('First', 'Last', 'First Last', 'First'),
+    ('First', 'Last', 'First Middle Last', 'First Middle'),
```

Comment:
This expected result is the worst thing we could do, because it removes the data we need entirely. Deleting a middle name is one thing, but it's gonna be hard to fix a CSV when you don't have the last name in it.